main.css

```diff
@@ -0,0 +1,634 @@
+  text-decoration: none;
+  color: white;
+}
+
+nav ul li a:hover {
+  background-color: #777;
+  border-radius: 10px;
+}
+
+.article-1 a:hover {
+  color: red;
+}
+
+nav ul li button {
+  margin-top: -7px;
+  padding: 8px 28px;
+  border-radius: 10px;
+  border: none;
+  color: white;
+  background-color: #333;
+  cursor: pointer;
+  font-size: 1.1em;
+  font-weight: bold;
+  font-family: "Baloo Paaji 2", cursive;
+}
+
+.container-article button {
+  color: rgb(44, 135, 165);
+  background-color: white;
+  border: 0;
+  border-radius: 10px;
+  margin-top: 5%;
+  padding: 25px 50px;
+  font-size: 1.5em;
+  font-weight: bold;
+}
+
+.grid-6 h1 {
+  font-size: 3em;
+  text-decoration: underline;
+  cursor: pointer;
+  line-height: 50px;
+}
+
+.grid-5 h1 {
+  font-size: 1.7em;
+  margin-left: 15%;
+  margin-top: -8%;
+  width: 100%;
+}
+
+.container-article-2 a h1 {
+  color: white;
+  font-size: 3em;
+  text-decoration: underline;
+  width: 60%;
+  margin: 2% 5%;
+  line-height: 50px;
+}
+
+.grid-6 h5 {
+  word-spacing: 5px;
+  font-size: 1.1em;
+  font-family: "Times New Roman", Times, serif;
+  color: #777;
+}
+
+footer p {
+  text-align: center;
+  color: white;
+  font-size: 22px;
+  line-height: 35px;
+}
+
+.grid-6 p {
+  color: #777;
+  font-size: 1.1em;
+}
+
+.grid-6 button {
+  color: white;
+  border: 0;
+  border-radius: 10px;
+  margin-top: 5%;
+  padding: 20px 50px;
+  font-size: 1.5em;
+  font-weight: bold;
+}
+
+.article-header h5 span {
+  text-decoration: none;
+  font-weight: lighter;
+  font-style: oblique;
+}
+
+.article-body p span {
+  font-size: 0.4em;
+}
+
+.container-article h5 {
+  font-size: 1.2em;
+  margin: 0;
+  padding: 0;
+}
+
+.container-article p {
+  font-size: 1.8em;
+  width: 70%;
+  margin: -2% auto 0 auto;
+}
+
+.grid-4 p {
+  font-size: 1.2em;
+  margin-top: 0;
+  margin-left: -10%;
+  max-width: 100%;
+}
+
+.article-header-2 h5 {
+  font-size: 1.5em;
+  margin-top: -1%;
+}
+
+.grid-5 p {
+  width: 90%;
+  font-size: 1.3em;
+  line-height: 30px;
+}
+
+.container-article-2 h5 {
+  font-size: 1.2em;
+  margin: 0 5%;
+  padding: 0;
+}
+
+.container-article-2 button {
+  color: rgb(44, 135, 165);
+  background-color: white;
+  border: 0;
+  border-radius: 10px;
+  padding: 20px 35px;
+  font-size: 1.5em;
+  font-weight: bold;
+  margin: 0 0 5% 5%;
+}
+
+.article-header h5 {
+  margin-top: 2%;
+  font-size: 1.2em;
+}
+
+.article-body p {
+  width: 80%;
+  font-size: 3em;
+  line-height: 50px;
+  text-transform: capitalize;
+  margin-right: 0;
+}
+
+.container-article-2 p {
+  font-size: 1.2em;
+  width: 80%;
+  margin: 2% 5%;
+  line-height: 40px;
+  font-family: "Times New Roman", Times, serif;
+}
+
+.grid-6 article,
+.grid-6 article h5,
+.grid-6 article h1,
+.grid-6 article p,
+.grid-6 article button {
+  margin: 5% 5%;
+}
+
+.container-article a h1 {
+  color: white;
+  font-size: 3em;
+  text-decoration: underline;
+  width: 60%;
+  margin: 5% auto;
+  line-height: 50px;
+}
+
+.grid-5 h1:hover {
```

Comment:
This selector has the same styles of selectors on line 288 and line 293. Don't repeat yourself, in order to minimize code you can group selectors with the same style definition.